packages/top-app-bar/index.js

```diff
@@ -48,11 +48,17 @@ export default class TopAppBar extends React.Component {
     this.foundation_.init();
   }
 
-  addClassesToElement(classes, element) {
-    const propsWithClasses = {
+  addClassesAndEnableRippleOnElement(classes, element) {
```

Comment:
methods are more likely to have bugs if they have the word "and" in their name.

Split this into two methods.

Reply:
I believe that...where'd you hear that?